modules/users/client/directives/password-validator.client.directive.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+angular.module('users')
+  .directive('passwordValidator', ['$window', function($window) {
+    return {
+      require: 'ngModel',
+      link: function(scope, element, attrs, modelCtrl) {
+        modelCtrl.$parsers.unshift(function (password) {
+          var owaspPasswordStrengthTest = $window.owaspPasswordStrengthTest;
```

Comment:
Can we have a client service for `owasp`? Some user's might want to extend it, or have a bit more control over it's behavior.